container/docker/handler.go

```diff
@@ -167,7 +167,9 @@ func libcontainerConfigToContainerSpec(config *libcontainerConfigs.Config, mi *i
 	}
 	spec.Cpu.Mask = utils.FixCpuMask(config.Cgroups.CpusetCpus, mi.NumCores)
 
-	spec.HasNetwork = len(config.Networks) > 0
+	// Docker reports a loop device for containers with --net=host. Ignore
+	// those too.
+	spec.HasNetwork = len(config.Networks) > 1
```

Comment:
WDYT if we count the non-loopback devices? If in the future this changes we don't have to worry about it.